recipes/py2700/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "py2700" %}
+{% set version = "0.0.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py2700-{{ version }}.tar.gz
+  sha256: b957b64b63164939ef15d888b8f124592b631a0d814588cbe6b3f735131d547d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - pyvisa
+    - pyvisa-py
+
+test:
+  imports:
+    - py2700
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ngpaladi/py2700
+  summary: 'A Python package to interface with a Keithley 2700 multimeter'
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
I've opened a PR upstream to address manually packaging the license file:
https://github.com/ngpaladi/py2700/pull/1

Once it is merged or if some other solution is used and a new release is spun out, then the manually packaged license file should be removed from the feedstock.

Reply:
Thank you, I will merge that and create a new release promptly. I can see if an older version of pyvisa works as well, or just wait for the update to pyvisa.